website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -176,4 +176,64 @@ Inline alerts can be added to a section or component or inline with content.
 
 Compact alerts can be added to a section or component or inline with content.
 
-![Placement of compact Alert](/assets/components/alert/alert-placement-compact.png =600x*)
\ No newline at end of file
+![Placement of compact Alert](/assets/components/alert/alert-placement-compact.png =600x*)
+
+## Multiple alerts
+
+In general, displaying multiple alerts on the same page is discouraged as it can negatively impact the page hierarchy and unnecessarily shift emphasis away from the main content on the page.
+
+However, depending on the logic within the application and the type content being expressed to the user, it may be necessary to display more than one Alert on a page. Consider these scenarios and their accompanying guidelines if this is the case.
```

Comment:
```suggestion
However, depending on the logic within the application and the type of content being expressed to the user, it may be necessary to display more than one Alert on a page. Consider these scenarios and their accompanying guidelines if this is the case.
```